perf.groovy

```diff
@@ -32,10 +38,8 @@ def generate(boolean isPr) {
     Utilities.addArchival(myJob, archiveSettings)
     Utilities.standardJobSetup(myJob, projectName, isPr, defaultBranch)
     Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-perf')
-    Utilities.addGithubPushTrigger(myJob)
```

Comment:
I believe that you only need a push trigger for a non-PR job, which is added in the else statement below. So this one is a duplicate.

@mmitche  Is this the case?